data/human/free worlds war jobs.txt

```diff
@@ -15,6 +15,37 @@ phrase "free worlds scanner payment"
 	word
 		`A Free Worlds captain is waiting for you when you land and takes a copy of your scanner logs, and you collect your payment of <payment>.`
 
+trade
+	commodity "free worlds outfits"
```

Comment:
If the outfit is a specific thing you can buy, it should be capitalized. For example, `breeder reactors` should be `Breeder Reactors`.

Reply:
I think I will remove it since I want to represent general commodities rather than specific ones, and breeder reactors come under fission reactors.